torch/_inductor/utils.py

```diff
@@ -792,9 +793,9 @@ def get_first_incompatible_cudagraph_node(
 
     for node in gm.graph.nodes:
         if str(node.target) in forbidden_set:
-            return node
+            return node, "incompatible op"
         if (val := node.meta.get("val")) is not None and free_unbacked_symbols(val):
```

Comment:
We can refine this check to :

a) skip unbacked symbols on inputs 
b) ignore unbacked symbols that were not newly introduced, that is were present on the node inputs